package.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "catberry-ts",
+    "version": "0.0.0",
+    "description": "TypeScript definitions for Catberry",
+    "author": "Friedrich von Never",
+    "contributors": [{
+        "name": "Alexey Kutepov a.k.a. rexim",
+        "email": "reximkut@gmail.com"
+    }],
+    "dependencies": {
+        "shelljs": "0.3.0",
+        "jsdoc": "3.3.0",
+        "xml2js": "0.4.8",
+        "es6-promise": "2.1.1"
+    },
+    "engines": {
+        "node": ">=0.10"
+    },
+    "scripts": {
+        "compile": "tsd reinstall -s && gulp typescript",
+        "jsdoc": "gulp jsdoc",
+        "tsdef": "gulp tsdef"
+    },
+    "devDependencies": {
+        "gulp": "gulpjs/gulp",
+        "gulp-typescript": "^2.7.5",
+        "typescript": "1.4",
+        "tsd": "0.6.0",
+        "typescript-register": "ForNeVeR/typescript-register#js"
```

Comment:
We will experiment with this later. I think that the official `1.1.0` is enough and that my fork should be purged.